### PR TITLE
ci: don't use separate config for mobile e2e

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

### CODE REVIEW

File: projects/demo-integrations/cypress/plugins/index.ts [hunk 1]
```diff
@@ -1,9 +1,11 @@
 // don't change import to `@taiga-ui/testing/cypress` until merging of this PR
 // https://github.com/jaredpalmer/cypress-image-snapshot/pull/250
```
Comment (nsbarsukov) [inline projects/demo-integrations/cypress/plugins/index.ts:2]: Put this comment before

```ts
import {tuiAddSnapshotPlugin} from '@taiga-ui/testing/cypress/snapshot/plugin';
```

File: projects/demo-integrations/cypress/support/helpers/get-by-example-id.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export function tuiEnsureExampleId(): string {
+    return Cypress.env(`EXAMPLE_ID`) ?? `tui-doc-example`;
+}
+
+export function tuiGetByExampleId(): Cypress.Chainable<unknown> {
```
Comment (nsbarsukov) [inline projects/demo-integrations/cypress/support/helpers/get-by-example-id.ts:5]: This naming of function supposes that the first argument is `exampleId` :think:

Maybe we should rename them to `tuiEnsureDocExample()`, `tuiGetDocExample()`, `tuiFindDocExample()` ?

**Review by nsbarsukov — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/demo-integrations/cypress/support/helpers/get-by-example-id.ts (hunk 2 above)
Comment (splincode) [reply projects/demo-integrations/cypress/support/helpers/get-by-example-id.ts]: fixed

File: projects/demo-integrations/cypress/plugins/index.ts (hunk 1 above)
Comment (splincode) [reply projects/demo-integrations/cypress/plugins/index.ts]: fixed

**Pull request merged**